### PR TITLE
Update herdsman debug log section for Home Assistant OS

### PR DESCRIPTION
**Pull request by ajfriesen**
Update the instructions.
We now have a button available and not configuration text. Also add a deep link to the zigbee2mqtt addon

### CODE REVIEW

File: docs/guide/usage/debug.md [hunk 1]
```diff
@@ -24,21 +24,12 @@ To enable debug logging for Zigbee-herdman start Zigbee2MQTT with: `DEBUG=zigbee
...
-  `zigbee_herdsman_debug: true`
+- Toggle the option: `zigbee_herdsman_debug`
 - Click `Save`, and when prompted to restart, click `Restart add-on`
 
-Herdsman debug logs should now sow up on the `Logs` tab for the addon.
-
-To grab all the logs, log in via SSH and execute:
-
-```
-docker logs CONTAINER_ID > log.txt 2>&1
```
[the remark this answers is not in the document]
Comment (ajfriesen) [reply docs/guide/usage/debug.md:38]: Yep, just checked and you are right.
The Log tab will not show all of the logs after some time.

File: docs/guide/usage/debug.md [hunk 2]
```diff
@@ -24,21 +24,18 @@ To enable debug logging for Zigbee-herdman start Zigbee2MQTT with: `DEBUG=zigbee
...
-Herdsman debug logs should now sow up on the `Logs` tab for the addon.
+Herdsman debug logs should now show up on the `Logs` tab for the addon.
 
 To grab all the logs, log in via SSH and execute:
 
 ```
 docker logs CONTAINER_ID > log.txt 2>&1
 ```
 
-To determine the `CONTAINER_ID` execute `docker ps`.
```
Comment (Koenkk) [inline docs/guide/usage/debug.md:41]: This line is also needed

Comment (ajfriesen) [reply docs/guide/usage/debug.md]: Good point. Obvious to me but not for everybody!

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**